src/Plugin/InlineEntityForm/CommerceProductInlineEntityFormController.php

```diff
@@ -1,29 +1,33 @@
 <?php
 
 /**
- * @file
- * Defines the inline entity form controller for Commerce Products.
+ * Contains \Drupal\inline_entity_form\Plugin\InlineEntityForm\CommerceProductInlineEntityFormController.
  */
 
+namespace Drupal\inline_entity_form\Plugin\InlineEntityForm;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+
```

Comment:
Likely on your radar but the factory class for this plugin could be cleaned up by checking for this interface and also by declaring an annotation for the type.